File: GymCRM.SchedulingAPI/Models/DTOs/InsertDailyAvailability.cs
namespace GymCRM.SchedulingAPI.Models.DTOs;

public class InsertDailyAvailability
{
    public string DayOfWeek { get; set; }
    public bool IsDayOff { get; set; }
    public List<InsertWorkingHours> WorkingHours { get; set; }
}

File: gymcrm.web/src/modules/scheduling/types/insertDailyAvailability.ts
import {InsertWorkingHours} from "./insertWorkingHours";

// Mirrors GymCRM.SchedulingAPI.Models.DTOs.InsertDailyAvailability 1:1.
export interface InsertDailyAvailability {
    dayOfWeek: string;
    isDayOff: boolean;
    workingHours: InsertWorkingHours[];
}
